BUILD.md

```diff
@@ -2,17 +2,29 @@
 
 This document contains instructions for producing private builds of tensorflow-directml.
 
-## DirectML and DirectML-Dev Branches
+## DirectML Branch
 
 This project is a fork of the official [tensorflow](https://github.com/tensorflow/tensorflow) repository that targets TensorFlow v1.15. This fork will not merge upstream, since TensorFlow does not accept new features for previous releases, so the `master` branch is based off v1.15.0 from the upstream repository. DirectML changes do not appear in the `master` branch.
 
-The `directml` branch is considered the main branch for development in this repository, and it should be the most stable branch to test. Packages of [tensorflow-directml on PyPI](https://pypi.org/project/tensorflow-directml/) are produced from the `directml` branch. This branch contains changes related to DirectML as well as security fixes in the upstream branch (e.g. changes in 1.15.1, 1.15.2, 1.15.3., etc.).
-
-The latest changes occur in the `directml-dev` branch. This branch is intended for rapid development, so builds of the `directml-dev` branch may reference unstable DirectML APIs that are subject to change over time; preview DirectML APIs are included in a special preview redistributable of DirectML that requires *developer mode*. 
+The `directml` branch is considered the main branch for development in this repository. This branch contains changes related to DirectML as well as security fixes in the upstream repository (e.g. changes in 1.15.1, 1.15.2, 1.15.3., etc.). Since this project is still in early development, the `directml` branch may be unstable and reference a "preview" version of DirectML that is subject to change. We recommend testing pre-built packages of [tensorflow-directml on PyPI](https://pypi.org/project/tensorflow-directml/); these builds are produced from the `directml` branch, but **only** when a stable version of DirectML is referenced. Alternatively, we recommend building against release tags associated with PyPI releases.
 
 ## Developer Mode
 
-**The python packages produced from the `directml-dev` branch will not run unless you enable developer mode, since they are not intended to be shipped or distributed for mainstream use**. To enable developer mode:
+Changes in this repository may reference in-development or preview versions of DirectML that are subject to change. This allows testing new APIs before they are committed to the DirectML API (once an API appears in DirectML.h it is immutable). A preview build of DirectML requires *developer mode* to be enabled or it will fail to load; this restriction is intended to avoid a long-term dependency on the preview library. **Packages on PyPI will only be released when the repository can depend on a stable version of DirectML.**
```

Comment:
Maybe we should clarify that APIs that appear in the dev versions of the redist are not actually immutable (otherwise, we wouldn't be able to test them before committing them).